### PR TITLE
Feature - Autocomplete

### PR DESCRIPTION
**Pull request by OloruntobiAwoderu**
# Description
Added Location and autocomplete to Register
Add attribution link to Readme and code

## Type of change
New Feature

Please delete options that are not relevant.

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## Change status

- [x] Complete, tested, ready to review and merge
- [ ] Complete, but not tested (may need new tests)
- [ ] Incomplete/work-in-progress, PR is for discussion/feedback

# How Has This Been Tested?

Please describe steps taken to ensure this feature is functional and does not break other features:

# Checklist:

- [x] My code follows the style guidelines of this project
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation, if relevant
- [x] I have run the app using my feature and ensured that no functionality is broken
- [x] My changes generate no new warnings
- [x] There are no merge conflicts

### CODE REVIEW

File: src/helpers/Error.js [hunk 1]
```diff
@@ -7,7 +7,7 @@ const Error = ({ touched, message }) => {
   if (message) {
   return <div className='form-message invalid'>{message}</div>
   }
-  return <div className='form-message valid'>all good</div>
+  return <div className='form-message valid'> &#9989;</div>
```
Comment (iamcristos) [inline src/helpers/Error.js]: whats this supposed to be?

File: package.json [hunk 2]
```diff
@@ -6,11 +6,14 @@
     "@testing-library/jest-dom": "^4.2.4",
     "@testing-library/react": "^9.3.3",
     "@testing-library/user-event": "^7.1.2",
+    "algolia-places-react": "^1.5.1",
     "axios": "^0.19.0",
+    "dotenv": "^8.2.0",
```
Comment (iamcristos) [inline package.json]: what exactly do we need dotenv for?

Comment (OloruntobiAwoderu) [reply package.json]: For the algolia app id and secrets. Setting them to the .env file

File: src/helpers/Error.js (hunk 1 above)
Comment (OloruntobiAwoderu) [reply src/helpers/Error.js]: This is what it looks like  ✅

File: package.json (hunk 2 above)
Comment (iamcristos) [reply package.json]: but we don't need dotenv, simply use REACT_APP_APP_ID and REACT_APP_APP_KEY

File: src/helpers/Error.js (hunk 1 above)
Comment (iamcristos) [reply src/helpers/Error.js]: cool nice

**Pull request merged**